src/user/user.service.ts

```
@@ -1,4 +1,60 @@
+        if (error.code === 'P2002')
+          throw new InternalServerErrorException('Personal information must not be duplicated');
+      throw new InternalServerErrorException();
+    });
+  }
+
+  update(id: number, updateUserDto: UpdateUserDto) {
+    return `This action updates a #${id} user`;
+  }
+
+  async deleteUser(uid: number) {
+    return await this.prisma.user.delete({
```

ESLint

[error] Insert `⏎······`

(prettier/prettier)